akka-actor-tests/src/test/scala/akka/serialization/NoVerification.scala

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package test.akka.serialization
+
+import akka.actor.NoSerializationVerificationNeeded
+
+/**
+ *  This is currently used in NoSerializationVerificationNeeded test cases in SerializeSpec,
+ *  as they needed a serializable class whose top package is not akka.
+ *
+ *  See a GitHub pull request <a href="https://github.com/akka/akka/pull/20785">#20785</a>
+ *  for more detail.
```

Comment:
OK sure, I followed your advice and removed the explicit reference in the new commit - thanks!